README.md

```diff
@@ -32,14 +32,17 @@ This is a simple shell script that can be used to ignore files from dropbox usin
 
 ### Basic Installation
 
-dropboxignore is installed by running one of the following commands in your terminal. You can install this via the command-line with either curl, wget or another similar tool. `attr` and `git` package should be installed on your system.
+dropboxignore is installed by running one of the following commands in your terminal. You can install this via the command-line with either curl, wget or another similar tool. `attr` and `git` package should be installed on your system, as well as the `diffutils` and `grep` packages for macOS.
 
 | Mathod | Command                                                                                                       |
 |--------|---------------------------------------------------------------------------------------------------------------|
 | curl   | `sudo sh -c "$(curl -fsSL https://raw.githubusercontent.com/sp1thas/dropboxignore/master/utils/install.sh)"`  |
 | wget   | `sudo sh -c "$(wget -qO- https://raw.githubusercontent.com/sp1thas/dropboxignore/master/utils/install.sh)"`   |
 | fetch  | `sudo sh -c "$(fetch -o - https://raw.githubusercontent.com/sp1thas/dropboxignore/master/utils/install.sh)"`  |
 
+### Installation notes for MacOS
```

Comment:
I propose to add these `brew install` commands directly into the `utils/install.sh` file.

Reply:
Sounds good - I forgot that users of dropboxignore are pretty much guaranteed to be developers so they will definitely have homebrew